api/serializers/phones.py

```diff
@@ -29,9 +29,12 @@ class RelayNumberSerializer(serializers.ModelSerializer):
     class Meta:
         model = RelayNumber
         fields = [
+            "id",
```

Comment:
Did you mean to expose the `id`?

Reply:
It looks like "yes", you need to know the ID so you can `PATCH` it later

Reply:
Yeah, exactly - I'm returning the `id` so it can be sent in the `PATCH`.